x/cosmwasmpool/genesis.go

```diff
@@ -26,7 +26,7 @@ func (k *Keeper) InitGenesis(ctx sdk.Context, gen *types.GenesisState, unpacker
 func (k Keeper) ExportGenesis(ctx sdk.Context) *types.GenesisState {
 	params := k.GetParams(ctx)
 
-	pools, err := k.GetPools(ctx)
+	pools, err := k.GetPoolsWithWasmKeeper(ctx)
```

Comment:
This should be `GetPools`, no reason to export the one with wasm keeper

Reply:
Same error occurs if I do not do this

Reply:
Disregard, was maybe cache or something, this works